$OpenBSD: patch-src_tools_clangpchmanagerbackend_source_collectbuilddependencypreprocessorcallbacks_h,v 1.1 2018/12/08 15:08:58 rsadowski Exp $

clang: error: non-virtual member function marked 'override' hides virtual member function

Index: src/tools/clangpchmanagerbackend/source/collectbuilddependencypreprocessorcallbacks.h
--- src/tools/clangpchmanagerbackend/source/collectbuilddependencypreprocessorcallbacks.h.orig
+++ src/tools/clangpchmanagerbackend/source/collectbuilddependencypreprocessorcallbacks.h
@@ -94,7 +94,7 @@ class CollectBuildDependencyPreprocessorCallbacks fina
                             llvm::StringRef /*searchPath*/,
                             llvm::StringRef /*relativePath*/,
                             const clang::Module * /*imported*/,
-                            clang::SrcMgr::CharacteristicKind fileType) override
+                            clang::SrcMgr::CharacteristicKind fileType)
     {
         if (file) {
             addSourceDependency(file, hashLocation);
